### ISSUE TITLE
Poor UI

### ISSUE THREAD

**Issue by olivierobert** [m-rwash/google-scraping#8]
While no design was provided, we do expect you to pay attention to the finished product so that it still does look neat and finished.

![googlescraping](https://user-images.githubusercontent.com/696529/34552445-0bbddcf8-f154-11e7-9abb-f9376cb2e6d4.png)

![googlescraping](https://user-images.githubusercontent.com/696529/34552467-2ca1c934-f154-11e7-9328-b5152dc99154.png)

![googlescraping](https://user-images.githubusercontent.com/696529/34552603-fd2e1940-f154-11e7-9e58-140ae865d9d3.png)

**Comment by m-rwash** [comment]
Yes I need to work on the UI more, I was just focusing on the backend
